client/src/components/Verejne/Map/ClusterMarker/index.js

```diff
@@ -32,20 +36,23 @@ const ClusterMarker = ({
   onClick,
   openedAddressId,
 }: ClusterMarkerProps) => {
-  const MarkerText = <span className="marker__text">{cluster.numPoints}</span>
+  const MarkerText = <span className='marker__text'>{cluster.numPoints}</span>
   let className, children
-  const selected = cluster.numPoints === 1 && cluster.points[0].address_id === openedAddressId
+  const selected =
+    cluster.numPoints === 1 && cluster.points[0].address_id === openedAddressId
   if (zoom < ENTITY_ZOOM) {
     className = cluster.numPoints === 1 ? 'simple-marker' : 'cluster-marker'
-    children = cluster.numPoints !== 1 && <span className="marker__text">{cluster.numPoints}</span>
+    children = cluster.numPoints !== 1 && (
+      <span className='marker__text'>{cluster.numPoints}</span>
+    )
   } else {
     //TODO: fix classnames after the api provides enough information
     className = classnames({
-      "company-marker": cluster.numPoints === 1,
-      "cluster-marker": cluster.numPoints > 1,
-      government: cluster.points[0].tradewithgovernment
-    });
-    children = cluster.numPoints === 1 ? <FaIconCircle size="18" /> : MarkerText
+      'company-marker': cluster.numPoints === 1,
+      'cluster-marker': cluster.numPoints > 1,
+      government: cluster.points[0].tradewithgovernment,
+    })
+    children = cluster.numPoints === 1 ? <FaIconCircle size='18' /> : MarkerText
```

Comment:
Is your formatter creating `size='18'` instead of `size="18"` ?
Is it also not quoting `government` ? If yes, we must fix your setup.

Reply:
Actually, i did that by hand, confused by the single quote bias.
But finally, i configured eslint to run in my editor with Vacuumlabs config :sun_with_face: